imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
+  -File (((Get-UninstallRegistryKey 'Image Resizer for Windows').UninstallString -split '"')[1])
```

Comment:
I do not know if the display name can be localized. This is the accepted solution by the community. MSIs are easier, they can be inspected for a unique `ProductCode`, which can be provided to Chocolatey and it handles the rest. These other installer types just register some info in the registry. We only have the `DisplayName` to go on :(